server/borsen/io/feed_parsers/ritzau.py

```diff
@@ -87,10 +87,22 @@ def find_similar_article(self, headline):
 
         try:
             cutoff = datetime.utcnow() - timedelta(hours=24)
-            ingest_items = superdesk.get_resource_service("ingest").get(
-                req=None, lookup={"headline": {"$exists": True}, "_created": {"$gte": cutoff}}
-            )
-            for item in ingest_items:
+            query = {
+                "headline": headline,
+                "_created": {"$gte": cutoff},
+                "more_like_this": {
+                    "min_term_freq": 1,
+                    "max_query_terms": 25,
+                    "min_doc_freq": 1,
+                    "minimum_should_match": "80%",
+                },
+            }
+
+            ingest_service = superdesk.get_resource_service("ingest")
+            results = ingest_service.get(req=None, lookup=query)
```

Comment:
would be probably better with `.search()` using source param:
```py
source = {
    "query": {
         "more_like_this": {
            "fields": ["headline"],
            "like": headline,
            "min_doc_freq": 1,
            "max_query_terms": 20,
            "minimum_should_match": "80%",
        }
    }
}

 results = ingest_sevice.search(source) 
```
probably with that cutoff combined in a bool query